src/Views/batchprocessor.html

```diff
@@ -104,18 +156,35 @@ <h3>Select Basin Characteristics</h3>
                         </tr>
                     </tbody>
                 </table>
+            </div><!-- End parameter selection -->
+
+        </div><!-- End submit batch -->
+        
+
+        <!-- Start batch status -->
+        <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
+            <p>This is the batch status tab</p>
+        </div> <!-- End batch status -->
+       
+</div> <!-- End of modal body -->
+
+<!-- Start modal footer -->
+<div class="modal-footer">
+    <div class="row" style="justify-content: center;">
+        <!-- Button to submit batch -->
+        <button ng-if="vm.selectedBatchProcessorTabName==='submitBatch'" type="submit" class="btn-wim-default" style="margin-right: 10px;" ng-click="vm.submitBatch()"
+            ng-disabled="vm.submittingBatch || !vm.selectedRegion || !vm.submitBatchData.idField || !vm.submitBatchData.email || !vm.submitBatchData.attachment"><i
 
-            </div>
-            <!-- End submit batch -->
+                class="fa fa-share"></i><span>&nbsp;&nbsp;Submit Batch</span></button>
+        <!-- End submit batch button -->
 
-            <!-- Start batch status -->
-            <div id="batchStatus" ng-if="vm.selectedBatchProcessorTabName==='batchStatus'">
-                <p>This is the batch status tab</p>
-            </div>
-            <!-- End batch status -->
-        </div>
+        <!-- Button to close modal -->
+        <button ng-click="vm.Close()" class="btn-black><i

```

Comment:
I noticed that the Close button wasn't black anymore

```suggestion
        <button ng-click="vm.Close()" class="btn-black"><i

```